sdk/samples/core/mosquitto_multiple_rpc_sample.c

```diff
@@ -0,0 +1,629 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file
+ * @brief RPC Server & Client sample for Mosquitto MQTT that supports multiple commands under the
+ * same subscription
+ *
+ */
+
+#include <az_log_listener.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <uuid/uuid.h>
+
+#include "rpc_client_pending_commands.h"
+#include <azure/az_core.h>
+#include <azure/core/az_log.h>
+#include <azure/core/az_mqtt5_rpc.h>
+#include <azure/core/az_mqtt5_rpc_client.h>
+
+#ifdef _WIN32
+// Required for Sleep(DWORD)
+#include <Windows.h>
+#else
+// Required for sleep(unsigned int)
+#include <unistd.h>
+#endif
+
+// User-defined parameters
+static const az_span client_id = AZ_SPAN_LITERAL_FROM_STR("application-id");
+static const az_span username = AZ_SPAN_LITERAL_FROM_STR("application-id");
+static const az_span hostname = AZ_SPAN_LITERAL_FROM_STR("localhost");
+static const az_span server_client_id = AZ_SPAN_LITERAL_FROM_STR("controller-id");
+static const az_span content_type = AZ_SPAN_LITERAL_FROM_STR("application/json");
+static const az_span start_module_command_name = AZ_SPAN_LITERAL_FROM_STR("startModule");
+static const az_span stop_module_command_name = AZ_SPAN_LITERAL_FROM_STR("stopModule");
+static const az_span server_subscription_topic_format
```

Comment:
Do we require customer to pass the string null-terminator? Can't we avoid that?

Reply:
Fixed! We now add the null-terminator in the topic generation